docs/docs/sdk/importing-model-predictions.mdx

```diff
@@ -80,18 +112,20 @@ That is:
 - `h`: box pixel height / image height
 
 ```python
-from encord_active.lib.db.predictions import BoundingBox, Prediction, Format
+from encord_active.lib.db.predictions import BoundingBox, Prediction, Format, ObjectDetection
 
 prediction = Prediction(
-    data_hash =  "<your_data_hash>",
+    data_hash="<your_data_hash>",
     frame = 3,  # optional frame for videos
-    class_id = "<the_class_id>",
     confidence = 0.8,
-    format = Format.BOUNDING_BOX,
-    # highlight-start
-    # Your bounding box coordinates in relative terms (% of image width/height).
-    data = BoundingBox(x=0.2, y=0.3, w=0.1, h=0.4)
-    # highlight-end
+    object=ObjectDetection(
+        object_class_hash="<the_class_id>",
```

Comment:
`object_class_hash` as well as `classification_hash` could be very confusing as the entry in the ontology is `featureNodeHash` and in the objects `featureHash` so perhaps just call it feature_hash or something completely new?

Reply:
Good point, I'll rename that